src/test/java/com/redhat/digkins/services/ArtifactsServiceTests.java

```diff
@@ -0,0 +1,86 @@
+package com.redhat.digkins.services;
+
+import com.offbytwo.jenkins.JenkinsServer;
+import com.offbytwo.jenkins.model.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Matchers;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ArtifactsServiceTests {
+
+  @Mock
+  private JenkinsServer server = mock(JenkinsServer.class);
+  private ArtifactsService artifactsService;
+
+  @Before
+  public void beforeTests() {
+    artifactsService = new ArtifactsService();
+  }
+
+  @Test
+  public void shouldFetchArtifactsWithNoResults() throws Exception {
+    JobWithDetails job = mock(JobWithDetails.class);
```

Comment:
I would gather all mocked objects together, below @Mock annotation or at least at the top of the test.